bin/fogbow-create-reverse-tunnel

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
```

Comment:
Removing fogbow?

Reply:
yep, @paulofelipefeitosa instead of fogbow-create-reverse-tunnel use create-reverse-tunnel There is no need to indicate it is a fogbow script in its name